main.cpp

```diff
@@ -78,44 +86,85 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 
 /*
  1)
- */
-
+ */
+ bool rentAHouse(int houseType, bool job = true)
+ {
+    ignoreUnused(houseType, job);
+    return {};
+ }
 /*
  2)
- */
+ */
+ float scienceClassNumbers(int students, bool approved, int tableNumber = 20)
+ {
+     ignoreUnused(students, approved, tableNumber);
+     return {};
+ }

```

Comment:
This function is not lined up with the Indentation guidelines in Replit.
Align your code with the vertical lines
- [ ] revise